src/popup/lib/utils.ts

```diff
@@ -62,3 +64,59 @@ export function formatNumber(
     } else return value.toExponential()
   }
 }
+
+export function useLocalStorage<T>(
```

Comment:
If this util is judged useful, I can move this into a separate PR.

Reply:
Yes. Let's move this in a separate PR.

Reply:
Sent https://github.com/interledger/web-monetization-extension/pull/419